containers/Auth/LoginScreen.styles.js

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '../../theme/colors';
+import { SIZES, FONTWEIGHT } from '../../theme/fonts';
+import { SPACINGS } from '../../theme/spacings';
+
+export const styles = StyleSheet.create({
+  flexContainer: {
+    flex: 1,
+    justifyContent: 'space-between',
+    backgroundColor: COLORS.paper.main,
+  },
+  container: {
+    paddingHorizontal: SPACINGS.multiplyBy(.75),
+  },
+  title: {
+    marginTop: SPACINGS.multiplyBy(4),
+    marginBottom: SPACINGS.multiplyBy(0.75),
+    fontSize: SIZES.xl,
+    fontWeight: FONTWEIGHT.boldxl
+  },
```

Comment:
it would be good if all of these could be defined once and just imported but that's for another day I think, along with the font clean ups

Reply:
Yeah, we do still have a bit of a mess with the styles. :woman_shrugging: